plugins/woocommerce/src/Admin/API/OnboardingTasks.php

```diff
@@ -762,6 +762,7 @@ public function dismiss_task( $request ) {
 
 		if ( ! $task && $id ) {
 			$task = new DeprecatedExtendedTask(
+				null,
```

Comment:
The constructor for `DeprecatedExtendedTask` [receives as first argument a "task list", not the task args](https://github.com/woocommerce/woocommerce/blob/70f464040e4b1ae43fe936fcdf184c3970fe1ac9/plugins/woocommerce/src/Admin/Features/OnboardingTasks/DeprecatedExtendedTask.php#L88-L89).

I'm not sure if this codepath is still triggered, but better safe than sorry.